app/src/sys/unix/command.c

```diff
@@ -1,24 +1,66 @@
+#define _GNU_SOURCE
```

Comment:
Oh, I didn't realize that `pipe2()` required this. This would make _scrcpy_ non portable (it would not compile on some platforms, possibly including Mac OS X (?)).

So we need `pipe()`+`fcntl()` instead (cf below).

Reply:
Will do.